dashboard/fixtures/certificates.json

```diff
@@ -5,14 +5,14 @@
 	    "certificate_type": "verified",
 	    "status": "downloadable",
 	    "download_url": "http://www.example.com/demo.pdf",
-	    "grade": "0.98"
+	    "grade": 98
```

Comment:
Elasticsearch limitation: https://discuss.elastic.co/t/less-than-1-interval-values/55201

we can't use decimal/float values for a histogram, which we need to construct the value ranges for the filter-by-grade component. 

as far as making it an int in the model vs. performing that in the serialization, i'm trying to remember exactly why i/we made that decision. can't find the hipchat convo right now, but off the top of my head, this would cause a disagreement between the data in our models and the data in Elasticsearch. i think we would need a compelling reason to introduce that disparity. i feel like i'm missing the real reason though - i'll dig through my notes and in hipchat